README.md

```diff
@@ -78,7 +78,7 @@ Rails/FindBy:
 
 ## Documentation
 
-You can read a lot more about RuboCop Rails in its [official docs](https://docs.rubocop.org/projects/rails/).
+You can read a lot more about RuboCop Rails in its [official docs](https://rails.rubystyle.guide/).
```

Comment:
It's actually https://docs.rubocop.org/rubocop-rails. Or at least - it will be, once we're done with our docs migration.